systemd/system.service

```diff
@@ -9,9 +9,27 @@ User=thelounge
 Group=thelounge
 Type=simple
 ExecStart=/usr/bin/thelounge start
+
+# Hardening
+LockPersonality=yes
 ProtectSystem=yes
+ProtectClock=yes
+ProtectControlGroups=yes
 ProtectHome=yes
+ProtectHostname=yes
+ProtectKernelLogs=yes
+ProtectKernelModules=yes
+ProtectKernelTunables=yes
+ProtectProc=invisible
+ProcSubset=pid
+RemoveIPC=yes
+# cacheable-lookup module requires AF_NETLINK
+RestrictAddressFamilies=AF_INET AF_INET6 AF_NETLINK
+RestrictNamespaces=yes
+RestrictRealtime=yes
+RestrictSUIDSGID=yes
```

Comment:
This prevents sgid on directories, which can be useful in certain circumstances (permissions basically, say for log files)
Not sure we really want to add that

Reply:
I am going to remove `RestrictSUIDSGID` to keep the retro-compatibility with users using SGID on directories.